### ISSUE TITLE
Update station status: Administrator

### ISSUE THREAD

**Issue by Avina20** [gcivil-nyu-org/wed-fall24-team4#3]
An admin should be able to promptly and accurately update any major changes related to accessibility features of subway stations, such as new ramps or broken elevators. He should be able to pin important posts, such as updates about station accessibility, temporary maintenance, or major accessibility feature disruptions, able to undo or delete any status updates if the information turns out to be false. He should also be able to verify the authenticity of any updates or reports by reviewing supporting documents or images and vetting the credibility of the users providing the information
